governance/board-sync.py

```
@@ -0,0 +1,354 @@
+                          f"State={c['state']} assignee={c['assignee'] or '-'}")
+                    stats["added"] += 1
+                    continue
+                item = gql(M_ADD_ITEM, {"i": {"projectID": pid, "contentID": c["node_id"]}})\
+                    ["addProjectV2ItemByContentId"]["item"]
+                entry = {"item_id": item["id"], "fields": {}}
+                stats["added"] += 1
+            want = {"Repo": c["repo"], "Assignee": c["assignee"] or "",
+                    "卡号": c["number"], "停留天数": c["days_idle"],
+                    "AC 进度": c["ac_progress"]}
+            have = entry["fields"]
+            # State 先比对（漂移报警面 = 宪法 §12 人工改动将被纠正）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)